Lib/test/test_urllib2.py

```diff
@@ -839,7 +838,7 @@ def test_file(self):
             try:
                 h.file_open(req)
             # XXXX remove OSError when bug fixed
```

Comment:
wow, it's right there in the context and still I missed it. Thanks for point out. I will remove it.